service/src/main/resources/application.yml

```diff
@@ -30,6 +30,7 @@ gpc-consumer:
     findPatientPath: ${GPC_CONSUMER_GPC_GET_PATIENT_PATH:/*/STU3/1/gpconnect/documents/fhir/Patient}
     # this is a common path for 'retrieve document' and 'migrate document' requests
```

Comment:
If we're sure about the difference between `documentPath` and `migrateDocumentPath`, this comment can be removed.